### ISSUE TITLE
ACC-287.1 (Level AA) _ Modify Modules _ Navigation menu _ Doesn’t satisfy Target size requirements

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3161]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	Navigation menu
Affected Check Point:	ACC-287.1
Short text:	ACC-287.1 (Level AA) _ Modify Modules _ Navigation menu _ Doesn’t satisfy Target size requirements 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Download ‘Minimum Target Size Check ’ tool and click on ‘Pick’ button from the tool.

3. Issue Type: Manual
Observed Behavior:
The menu item ‘Cluster Details’ from the left side navigation menu doesn’t satisfy the minimum target size requirements of 24x24 CSS pixels. 
Screenshot:
 

Expected Behavior:
The menu item ‘Cluster Details’ from the left side navigation menu should satisfy the minimum target size requirements of 24x24 CSS pixels.
Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.

________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-287.1 (Level AA) _ Modify Modules _ Navigation menu _ Doesn’t satisfy Target size requirements.docx](https://github.com/user-attachments/files/16660363/ACC-287.1.Level.AA._.Modify.Modules._.Navigation.menu._.Doesn.t.satisfy.Target.size.requirements.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16660364/Issue.processing-09.2021.002.docx)
